docs/en-US/component/anchor.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Anchor
+lang: en-US
+---
+
+# Anchor
+
+Through the anchor point, you can quickly find the position of the information content on the current page.
+
+## Basic Usage
+
+The most basic usage
+
+:::demo
+
+anchor/basic
+
+:::
+
+## Horizontal Mode
+
+Horizontally aligned anchors
+
+:::tip
+
+Horizontal Mode does not support `sub-link` slots
+
+:::
+
+:::demo
+
+anchor/horizontal
+
+:::
+
+## Scroll Container
+
+Custom scroll area, use `offset` props can set anchor scroll offset, listen the `link-click` event and prevents browser default behavior then it will not change history.
+
+:::demo
+
+anchor/scroll
+
+:::
+
+## Anchor link change
+
+Listening for anchor link change
+
+:::demo
+
+anchor/change
+
+:::
+
+## Underline type
+
+set `type="underline"` change to underline type
+
+:::demo
+
+anchor/underline
+
+:::
+
+## Affix Mode
+
+Use the affix component to fix the anchor point within the page.
+
+:::demo
+
+anchor/affix
+
+:::
+
+## Anchor API
+
+### Anchor Attributes
+
+| Property   | Description                              | Type                                                  | Default |
+| ---------- | ---------------------------------------- | ----------------------------------------------------- | ------- |
+| container  | scroll container.                        | `string` \| `HTMLElement` \| `Window `                | —       |
+| offset     | set the offset of the anchor scroll.     | `number`                                              | 0    |
+| bound      | the offset of the element starting to trigger the anchor. | `number`                             | 15      |
+| duration   | set the scroll duration of the container, in milliseconds.          | `number`                   | 300    |
+| marker     | Whether to show the marker.              | ^[boolean]                                            | true   |
```

Comment:
```suggestion
| marker     | whether to show the marker              | ^[boolean]                                            | true   |
```
We don't need uppercase letters or period in description, others too.

Reply:
done